core/src/com/threecubed/auber/AuberGame.java

```diff
@@ -11,7 +11,7 @@
   @Override
   public void create() {
     atlas = new TextureAtlas("auber.atlas");
-    Gdx.graphics.setWindowedMode(1920, 1080);
+    Gdx.graphics.setWindowedMode(1980, 1080);
```

Comment:
?

Reply:
My bad, I lowered it so it would fit on my screen, and typo